### ISSUE TITLE
Is TDD Dead? [30m]

### ISSUE THREAD

**Issue by nosamanuel** [CondeNast/atx-lunchnlearn#20]
A conversation with Martin Fowler, Kent Beck, and DHH. 

> "TDD as One True Way" versus "TDD as devil-spawned tempter" is not a productive contrast. Most of us have similar goals for development: confidence, impact, challenge, belonging. Test-driven development is one path toward these goals, but there are many other paths. This conversation in six parts explores tradeoffs in development, places TDD in those tradeoffs, and gives you perspective with which you can decide for yourself how you want to program.

https://www.youtube.com/watch?v=z9quxZsLcfo&t=4s&index=17&list=PLM5_23Q0ogsrsPP7cEmUtnAZNufdvTLCO (Part 1 of 4, 30:25)

**Comment by pgoldrbx** [comment]
Yes - there are always tradeoffs. I find that TDD tends to surface more bugs and unintended side-effects while giving me much greater confidence that my code is behaving as intended, but it takes more time to write and to maintain versus BDD. Also, BDD is more common in the node world given scoping of module exports versus "internal"/private methods.

**Comment by crccheck** [comment]
are we going to watch part 2?
